assets/js/qr_code_form/editor.js

```diff
@@ -1,20 +1,26 @@
 import Quill from 'quill'
 
+
 Quill.register('modules/counter', function(quill, options) {
-    var container = document.querySelector(options.container);
-    quill.on('text-change', function() {
-      var text = quill.getText();
+    const container = document.querySelector(options.container);
+    quill.on('text-change', () => {
+      const text = quill.getText();
       const maxCharacters = container.getAttribute('data-max-characters');
```

Comment:
sometimes I see semicolons and sometimes I don't. Better stay consistent or use a linter :)

Reply:
Yes, a linter for js as well sounds like the better option.